### PR TITLE
Added PM table entry for Zen 3 (key = 0x00380805 )

### PR DESCRIPTION
**Pull request by DevTechProfile**
Sorry, Master to Master. Was to lazy to create a specific branch for that small code adjustment.

### CODE REVIEW

**Comment by DevTechProfile** [comment]
Tested on my machine with Ryzen 9 5900X (fyi, 2 cores per CCD are disabled due to testing purposes). 

image

File: LibreHardwareMonitorLib/Hardware/RyzenSMU.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal class RyzenSMU
         private readonly Dictionary<uint, Dictionary<uint, SmuSensorType>> _supportedPmTableVersions = new()
         {
             {
-                // Zen Raven Ridge APU.
+                // Zen Raven Ridge APU
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/RyzenSMU.cs:27]: Undo.

File: LibreHardwareMonitorLib/Hardware/RyzenSMU.cs [hunk 2]
```diff
@@ -54,7 +54,7 @@ internal class RyzenSMU
                 }
             },
             {
-                // Zen 2.
+                // Zen 2
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/RyzenSMU.cs:57]: Undo.

File: LibreHardwareMonitorLib/Hardware/RyzenSMU.cs [hunk 3]
```diff
@@ -71,6 +71,25 @@ internal class RyzenSMU
                     //{ 192, new SmuSensorType { Name = "Core #5", Type = SensorType.Clock, Scale = 1000 } },
                     //{ 193, new SmuSensorType { Name = "Core #6", Type = SensorType.Clock, Scale = 1000 } },
                 }
+            },
+            {
+                // Zen 3
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/RyzenSMU.cs:76]: Also point + formatting below :)

**Review by PhyxionNL — CHANGES_REQUESTED** [review]
Thanks 👍 Values match with Ryzen Master?

**Comment by DevTechProfile** [comment]
> Thanks 👍 Values match with Ryzen Master?

No, it doesn't match the HWiNFO values.
image

**Comment by DevTechProfile** [comment]
Fabric, Uncore, Memory clocks are correct

**Comment by DevTechProfile** [comment]
> > Fabric, Uncore, Memory clocks are correct
> 
> It's likely that the other items are on a different index, you can try looping through the values and see if anything matches Ryzen Master. Keep in mind that the scale might be different 👍

Where do I get the scale from? Does the SMU itself provice these scales?

**Comment by PhyxionNL** [comment]
Trial and error :) But so far it always seems to be 1 or 1000.

**Comment by DevTechProfile** [comment]
> Trial and error :) But so far it always seems to be 1 or 1000.

Look at the screenshot. HWiNFO shows something like 12A and the SMU readout with scale = 1 looks like chaos. :-D

**Comment by PhyxionNL** [comment]
It's probably on a different index, I recommend looping through all of them and see which value matches with Ryzen Master or HWiNFO.

**Comment by DevTechProfile** [comment]
Gonna check this later this evening.

**Comment by PhyxionNL** [comment]
> Gonna check this later this evening.

Any update?

**Comment by DevTechProfile** [comment]
> > Gonna check this later this evening.
> 
> Any update?

Yes, indeed. I tried to identify the correct values in the array. Impossible because for CPU TDC e.g. there are 3 values that match with with the HWiNFO sensor. What makes it even more difficult to find a match is an update cycle offset. 

Is there any documentation to be able to identify the indices of the array?

The only entries which work properly are

`          
{ 48, new SmuSensorType { Name = "Fabric", Type = SensorType.Clock, Scale = 1 } },
{ 50, new SmuSensorType { Name = "Uncore", Type = SensorType.Clock, Scale = 1 } },
{ 51, new SmuSensorType { Name = "Memory", Type = SensorType.Clock, Scale = 1 } },
`

I will adjust the code and push an update later.

**Pull request closed without merging**